inc/WC_JumiaPay_Purchase.php

```diff
@@ -83,58 +71,60 @@ private function getBasket() {
     }
 
     public function generateData() {
+        $merchantReferenceId = $this->generateMerchantReference();
 
         return [
-            "shopConfig" => $this->shopConfig,
+            "description" => "Payment for order #" . $merchantReferenceId,
```

Comment:
please do not forget to sanitize and validate max length here too.

Reply:
done